### PR TITLE
Add feature contribution guidelines to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by bnaul**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,4 +8,11 @@ We gratefully accept contributions of new time-series features, be they
 domain-specific or general. Please follow the below guidelines in order that
 your features may be successfully incorporated into the Cesium feature base.
 
-*Coming soon*
+1. Add any new numerical code to a new or existing file `cesium/features/`.
```
Comment (acrellin) [inline CONTRIBUTING.md]: "in"

**Comment by acrellin** [comment]
Looks great :+1:

**Comment by acrellin** [comment]
This done @bnaul?

**Comment by bnaul** [comment]
Yeah as long as everyone thinks it sounds good I'm happy :+1:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,8 +4,11 @@
 
 ## Contributing Time-Series Features
 
-We gratefully accept contributions of new time-series features, be they
-domain-specific or general. Please follow the below guidelines in order that
-your features may be successfully incorporated into the Cesium feature base.
-
-*Coming soon*
+We gratefully accept contributions of new time-series features, be they domain-specific or general. Please follow the guidelines below so that your features are successfully incorporated into the `cesium` feature base.
```
Comment (stefanv) [inline CONTRIBUTING.md]: How about (2nd sentence).  "To contribute a new feature, follow these steps:" or "You may add a new feature as follows:"

**Comment by bnaul** [comment]
Replaced by #206.

**Pull request closed without merging**